lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -3350,7 +3351,7 @@ func TestAPNResource(t *testing.T) {
 		ParamNames:             []string{"network_id"},
 		ParamValues:            []string{"n0"},
 		ExpectedStatus:         500, // this would actually make more sense as a 400, but it's a non-trivial fix
-		ExpectedErrorSubstring: `could not find entities matching [type:"apn" key:"apn0" ]`,
+		ExpectedErrorSubstring: "could not find entities matching" + tests.Separator + "[type:\"apn\"" + tests.Separator + "key:\"apn0\"]",
```

Comment:
Nit: instead of escaping the quotes, use the backtick

Reply:
Fixed